python/hail/expr/functions.py

```diff
@@ -1401,7 +1401,9 @@ def binom_test(x, n, p, alternative: str) -> Float64Expression:
     :class:`.Expression` of type :py:data:`.tfloat64`
         p-value.
     """
-    return _func("binomTest", tfloat64, x, n, p, to_expr(alternative))
+
+    type = {"two.sided": 0, "greater": 1, "less": 2}[alternative]
```

Comment:
type is a bit overloaded, how about `alt_type`? (alt is also overloaded, but less so)